charts/fluid-dataloader/jindofsx/templates/configmap.yaml

```diff
@@ -78,19 +78,31 @@ data:
         else
             echo -e "--- disable loadMemorydata"
         fi
-
-        if [[ $enbaleCacheListLocation == 'false' ]]; then
-            cmd="$cmd -s -R -replica $replica $default$path"
-            echo -e "execute cmd $cmd"
-            time $cmd
+    
+        if [[ $enableDryrun == 'true' ]]; then
+            echo -e "--- enable dryrun"
+            cmd="$cmd -dryrun"
+            echo -e "now cmd $cmd"
         else
+            echo -e "--- disable dryrun"
+        fi
+
+        if [[ $enbaleCacheListLocation == 'true' ]]; then
             echo -e "--- begin download ossutil"
             apt-get install wget -y
             wget http://gosspublic.alicdn.com/ossutil/1.7.5/ossutil64
```

Comment:
Can we put ossutil64 into the docker image? Looks like it needs to internet access to download binary.